File: 0448 find-all-numbers-disappeared-in-an-array/solution.ts
function findDisappearedNumbers(nums: number[]): number[] {
    let set = [];
    let ans = [];
    for(let i=0;i<nums.length;i++) set[nums[i]]=true;
    for(let i=1;i<=nums.length;i++) if(!set[i]) ans.push(i);
    return ans;
};

File: 0448 find-all-numbers-disappeared-in-an-array/solution.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        vector<bool> arr(nums.size(),false);

        for(int i:nums) {
            arr[i-1] = true;
        }
        vector<int> ans;
        for(int i=1;i<=nums.size();i++) {
            if(!arr[i-1]) ans.push_back(i);
        }
        return ans;
    }
};
